### PR TITLE
chore: project addition by rishabhchaudhary0210 #1862

### PR DESCRIPTION
**Pull request by rishabhchaudhary0210**
## Description
Added The project -- Grocery Website made by me using HTML, CSS, JS, SwiperJS. It has a modern design and is fully responsive.

## Screenshots

image

image

### CODE REVIEW

File: DB/projects.json [hunk 1]
```diff
@@ -4304,5 +4304,21 @@
...
+    "LinkedIn": "https://www.linkedin.com/in/rishabh-chaudhary-108564154",
+    "Instagram": "https://instagram.com/_.rishabh.chaudhary._?igshid=MzNlNGNkZWQ4Mg==",
+    "Twitter": "https://twitter.com/ItsRishabh02?t=C_PAIivEq-r3hcAxAEURpg&s=08"
+  },
+  "Projects": [
+    {
+      "link": "https://github.com/rishabhchaudhary0210/Grocery-Website",
+      "title": "Groco-Grocery Website",
+      "description": "A website to fulfill your daily grocery needs. A static fronted designed to practice my skills and try out the Swiper.",
+      "tech": ["HTML", "CSS", "JavaScript", "SwiperJs"]
```
Comment (priyankarpal) [inline DB/projects.json:4320]: change to lowercase 
```suggestion
      "tech": ["html", "css", "javascript", "swiperjs"]
```

File: DB/projects.json [hunk 2]
```diff
@@ -4304,5 +4304,21 @@
...
+    },
```
Comment (priyankarpal) [inline DB/projects.json:4321]: remove the last `,`
```suggestion
    }
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: DB/projects.json (hunk 2 above)
Comment (rishabhchaudhary0210) [reply DB/projects.json]: done sir

File: DB/projects.json (hunk 1 above)
Comment (rishabhchaudhary0210) [reply DB/projects.json]: done sir

**Review by priyankarpal — APPROVED** [review]
Thank you for contributing your projects here 👏🏻 🎉

**Pull request merged**